lib/stringify.js

```diff
@@ -75,8 +77,23 @@ var stringify = function stringify(
 ) {
     var obj = object;
 
-    if (sideChannel.has(object)) {
-        throw new RangeError('Cyclic object value');
+    var tmpSc = sideChannel;
+    var step = 0;
+    var findFlag = false;
+    while ((tmpSc = tmpSc.get(scSym)) !== undefined && !findFlag) {
+        // Where object last appeared in the ref tree
+        var pos = tmpSc.get(object);
+        step += 1;
+        if (pos != undefined) {
```

Comment:
This should use !==, or != null; != should never be used to compare to anything besides null.

Reply:
i get it, thanks for the guidance.

Reply:
> 

i get it, thanks for the guidance.